composer.json

```diff
@@ -22,6 +22,7 @@
         "php-http/message": "^1.2",
         "psr/http-client": "^1.0",
         "psr/http-factory": "^1.0",
```

Comment:
should we drop this? we need the psr/http-client because we implement the interfaces and provide the http-client-implementation.

however, it seems safe to assume that every http-factory-implementation depends on the http-factory interfaces package... not sure what is better.